tests/PHPUnit/Fixtures/ThreeVisitsWithCustomEvents.php

```diff
@@ -60,7 +60,7 @@ public function trackVisits()
         $this->trackEventWithoutUrl($vis);
         $this->trackMovieWatchingIncludingInterval($vis);
 
-        $this->dateTime = Date::factory($this->dateTime)->addHour(0.5);
+        $this->dateTime = Date::factory($this->dateTime)->addHour(0.5)->getDatetime();
         $vis2 = self::getTracker($this->idSite, $this->dateTime, $useDefault = true, $uselocal);
```

Comment:
we used to pass a `Date` object here meaning the tracker would do `$date->toString()` which then would only result in `YYYY-MM-DD` and throw away the entire time. So this was before actually not working which is why some tests are failing.